Openshift-EE/pipelines/Cassandra/stages/2-setup/OJEF-create-striped-pool/create-pool

```diff
@@ -55,17 +72,18 @@ bash Openshift-EE/utils/event_updater jobname:cstor-pool $test_name jobid:"$job_
 #################
 rc_val=$(echo $?)
 
-
 ## Check the test status & result from the litmus result custom resource
 source ~/.profile
+
 gittoken=$(echo "$github_token")
+
 testResult=$(kubectl get litmusresult ${test_name} --no-headers -o custom-columns=:spec.testStatus.result)
-#python3 Openshift-EE/utils/result/result_update.py $job_id OJEF 2-setup "Create cStor based Striped storage pool on disks." $testResult $pipeline_id "$current_time" $commit_id $gittoken
+#python3 Openshift-EE/utils/result/result_update.py $job_id HC5P 2-setup "Create cStor based Striped storage pool on disks and corresponding storage class" $testResult $pipeline_id "$current_time" $commit_id $gittoken
```

Comment:
@w3aman please remove this commented lines.

Reply:
Done!